nemo/collections/nlp/modules/common/huggingface/bert.py

```diff
@@ -30,6 +30,6 @@ class BertEncoder(BertModel, BertModule):
     """
 
     @typecheck()
-    def forward(self, **kwargs):
-        res = super().forward(**kwargs)[0]
+    def forward(self, input_ids, attention_mask, token_type_ids):
+        res = super().forward(input_ids, attention_mask, token_type_ids)[0]
```

Comment:
please use named arguments

Reply:
actually @yzhang123  is right. I think our typecheck would throw en exception otherwise (and if it doesn't now - it could in the future)